apps/dune-simulator/src/endpoints/dune-simulator/dune-simulator.controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Body, Controller, Param, Post, Headers } from "@nestjs/common";
+import { ApiTags } from "@nestjs/swagger";
+import { CreateTableBody } from "./entities";
+import { DuneSimulatorService } from "@libs/services/dune-simulator";
+
+@Controller('/api/v1/table')
+@ApiTags('dune-simulator')
+export class DuneSimulatorController {
+    constructor(
+        private readonly duneSimulatorService: DuneSimulatorService,
+    ) { }
+
+    @Post("/table/create")
```

Comment:
the controller is defined as `/api/v1/table` and this endpoint as `/table/create`. Won't this result into `/api/v1/table/table/create` ?

Reply:
yes, you're right! I've just fix it in the next commit